bindgen-cli/options.rs

```diff
@@ -353,6 +353,21 @@ struct BindgenCommand {
     /// Derive custom traits on a `union`. The <CUSTOM> value must be of the shape <REGEX>=<DERIVE> where <DERIVE> is a coma-separated list of derive macros.
     #[arg(long, value_name = "CUSTOM")]
     with_derive_custom_union: Vec<String>,
+    /// Generate extern wrappers for inlined functions
+    #[arg(long, requires = "experimental")]
+    generate_extern_functions: bool,
```

Comment:
This seems like an unfortunate option name. I get that `generate_inline_functions` is taken, but generating extern functions is what bindgen does every time... Maybe just rename `extern_functions_file_name` to `inline_functions_file_name`, and use that to distinguish this new mode? Also probably worth combining `extern_functions_directory` and file_name into `extern_functions_path` or so?

Reply:
Oh I see this is not only about inline functions but also about non-external linkage functions. So maybe this should be called `generate_non_extern_functions` or so?